os/drivers/audio/alc5658.h

```diff
@@ -75,9 +75,14 @@
 #define FAIL						0xFFFF
 #define MUTE_HP_LR					0x8080
 #define UNMUTE_HP_LR				0x0
-#define IN1_PORT_GAIN				(0 + 16) << 8
 #define ALC5658_HP_VOL_MIN			0x0
 #define ALC5658_HP_VOL_MAX			0x1F
+#define ALC5658_HP_VOL_DEFAULT		0x1a
```

Comment:
I think all these value should be moved to c file.
Please do this later.

Reply:
Okay~.